src/RegexCollection/EmailAddress.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace YorCreative\Scrubber\RegexCollection;
+
+use YorCreative\Scrubber\Interfaces\RegexCollectionInterface;
+
+class EmailAddress implements RegexCollectionInterface
+{
+    public function getPattern(): string
+    {
+        return '[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}';
```

Comment:
👋🏼 Hey @JorgeAnzola  thanks for submitting this. 💪🏼 

I've taken a look at it, and overall it seems to be working well.

However, I noticed that the TLD is currently restricted to a maximum of four characters. While this may cover most common TLDs, there are now many new TLDs with longer extensions, such as .international, .photography, and .technology, that wouldn't be covered by the current regex.

To ensure the regex can detect all valid email addresses, I would ask that you expand the TLD to allow for up to 63 characters. This can be achieved by changing the {2,4} in the regex to {2,63}, as follows:

`[A-Z0-9._%+-]+@[A-Z0-9.-]+.[A-Z]{2,63}`

 [ https://regex101.com/r/ImkP4Y/1]( https://regex101.com/r/ImkP4Y/1)

Reply:
Also note you'll need to update +1 to the tests to account for this new core regex class. 
_See the failed tests for where you'll need to update._

Reply:
That's true! I'll push the changes

Reply:
💪🏼 Thanks for this @JorgeAnzola. Your time & contributions are much appreciated.  🙏🏼